### PR TITLE
Fixes bug where getBooleanInput is not available

### PR DESCRIPTION
**Pull request by retiman**
Last PR introduced some bugs.
- @actions/core needs to be bumped to 1.3.0 to support getBooleanInput.
- should be `trim()` and not `trim`.

Here's the link to a successful check run: https://github.com/retiman/read-file-action/runs/2674226609?check_suite_focus=true

Sorry about about the inconvenience.

### CODE REVIEW

**Review by Akhil-Sapaad — APPROVED** [review]

**Comment by Akhil-Sapaad** [comment]
@juliangruber can you merge this

**Comment by juliangruber** [comment]
> @juliangruber can you merge this

chill, this PR is just 4 minutes old :D

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const main = async () => {
   const trim = core.getBooleanInput('trim')
   let content = await fs.readFile(path, 'utf8')
   if (trim) {
-    content = content.trim
+    content = content.trim()
```
Comment (juliangruber) [inline index.js:11]: dang how did I miss this

**Comment by juliangruber** [comment]
https://github.com/juliangruber/read-file-action/releases/tag/v1.1.1

**Comment by Akhil-Sapaad** [comment]
Screenshot 2021-05-26 at 4 23 18 PM
 
 @juliangruber and @retiman please check this issue

**Comment by juliangruber** [comment]
do you have a link to that? it looks like you put `trim: trim`, not `trim: true`

**Pull request merged**